.github/workflows/docker-build-static.yml

```diff
@@ -39,9 +39,12 @@ env:
 # GitHub actions randomly timeout when downloading musl-gcc, using custom dev image   #
 # Look into .github/workflows/Dockerfile.dev for development image details            #
 # Using lldap dev image based on https://hub.docker.com/_/rust and musl-gcc bundled   #
+# lldap/rust-dev:latest                                                               #
 #######################################################################################
 ### Cargo build
-### aarch64 and amd64 is musl based
+### armv7, aarch64 and amd64 is musl based
+
+### This not an issue anymore when using rust 1.71 forward
 ### armv7 is glibc based, musl had issue with time_t when cross compile https://github.com/rust-lang/libc/issues/1848
```

Comment:
Should we update or remove this line?

Reply:
Im gonna remove this, to prevent confusion, as it isn't really an issue right now.